modules/ice/ice.c

```diff
@@ -383,7 +386,16 @@ static bool if_handler(const char *ifname, const struct sa *sa, void *arg)
 	if (!net_af_enabled(baresip_network(), sa_af(sa)))
 		return false;
 
-	lprio = 0;
+	lprio = m->lprio;
+
+	/* Check for default routes */
+	default6 = net_laddr_af(baresip_network(), AF_INET6);
+	if (sa_cmp(default6, sa, SA_ADDR))
+		lprio = UINT16_MAX;
+
+	default4 = net_laddr_af(baresip_network(), AF_INET);
+	if (sa_cmp(default4, sa, SA_ADDR))
+		lprio = UINT16_MAX - 1;
```

Comment:
perhaps it is enough to call net_laddr_af once here, the AF is coming as an input parameter
to if_handler via `sa`.